androgee/init.py

```diff
@@ -0,0 +1,22 @@
+import requests
+import sys
+import os
+
+global swear_list
+swear_list = requests.get(
+    "https://raw.githubusercontent.com/advaithm/badwords/master/badwords.json"
+).json()
+# I think it'd be better if we check if environment
+# variables are present before doing anything else.
+if os.path.exists(".env"):
+    from dotenv import load_dotenv
+
+    load_dotenv()
+try:
+    mod_role_id = os.environ["mod_role_id"]
```

Comment:
How come some are some use SCREAMING_SNAKE_CASE and others use regular snake_case?

Reply:
I'll change it